plugins/link/link.cpp

```diff
@@ -31,8 +31,8 @@ struct LinkedMem {
 	wchar_t description[2048];
 };
 
-static void about(HWND h) {
```

Comment:
Ooops! Indeed, longs are 32-bit on Windows x64.

How about we use void * as well?